From 7c0b4d02ef0154509b905a15272625a565c10bf2 Mon Sep 17 00:00:00 2001
From: Hades <15710513+hadesdevls@users.noreply.github.com>
Date: Sun, 11 Oct 2026 07:54:15 +0000
Subject: [PATCH] userdata: leave the news of LibreELEC out of the feeds

LibreELEC adds its own news to the feeds of the home screen, ahead of
Kodi's. On this build they are the news of another system: a release
of LibreELEC is not an update of it, and installing one would take the
Dolby Vision output away (which is why the updater does not look at
LibreELEC's server either). The file is Kodi's own again.

Only seen with the feeds turned on, which they are not by default. Any
feed can be added in the settings, as always.
---
 userdata/RssFeeds.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/userdata/RssFeeds.xml b/userdata/RssFeeds.xml
index 677c375..590a7c5 100644
--- a/userdata/RssFeeds.xml
+++ b/userdata/RssFeeds.xml
@@ -3,7 +3,6 @@
   <!-- RSS feeds. To have multiple feeds, just add a feed to the set. You can also have multiple sets. 	!-->
   <!-- To use different sets in your skin, each must be called from skin with a unique id.             	!-->
   <set id="1">
-    <feed updateinterval="30">https://libreelec.tv/feed</feed>
     <feed updateinterval="1440">https://kodi.tv/feed</feed>
   </set>
 </rssfeeds>
-- 
2.53.0

